src/airflow/airqo_etl_utils/airqo_api.py

```diff
@@ -232,25 +231,23 @@ def refresh_airqloud(self, airqloud_id):
         query_params = {"tenant": str(Tenant.AIRQO), "id": airqloud_id}
 
         try:
-            response = requests.put(
-                url=f"{self.AIRQO_BASE_URL_V2}/devices/airqlouds/refresh",
+            response = self.__request(
+                endpoint="devices/airqlouds/refresh",
                 params=query_params,
+                method="put",
             )
-
-            print(response.json())
         except Exception as ex:
             print(ex)
 
     def refresh_grid(self, grid_id):
         query_params = {"tenant": str(Tenant.AIRQO)}
 
         try:
-            response = requests.put(
-                url=f"{self.AIRQO_BASE_URL_V2}/devices/grids/refresh/{grid_id}",
+            response = self.__request(
+                endpoint=f"devices/grids/refresh/{grid_id}",
                 params=query_params,
```

Comment:
Hi @BenjaminSsempala , please review this logic.....could it be that we are using the AirQloud ID when refreshing Grids?